### ISSUE TITLE
Online demo doesn't work in Safari.

### ISSUE THREAD

**Issue by danscan** [madjam002/graphee#1]
Great repo!  The online demo works great in Chrome, but when I initially viewed it in Safari, no result was displayed for the query, and when I hit the Submit button, nothing happened.

**Comment by madjam002** [comment]
Cheers! Got quite a few local fixes which I need to push anyway, but I'll try and take a look at this tonight. Thanks for reporting :)

**Comment by danscan** [comment]
Definitely.  I'm literally refreshing a Twitter `graphql` search tab on a daily basis now– haha.  Happy to help out with this too, if you have TODOs.  

I'm working on a couple of private repos in es6 w/ es6 modules, and JS Standard style, so I like the style of this repo.

**Comment by madjam002** [comment]
Nice! Yeah I'm using this for an internal project so there's not really a TODO list unfortunately, I just work on it when I need to, but I should try and clean this up, improve the docs and get a roadmap sorted at some point.

**Comment by madjam002** [comment]
I've just taken a look at the official reference library which has just been released at https://github.com/graphql/graphql-js

I didn't expect for it to be a mapper as well! I'm considering not developing this library further now as it seems to do everything that I was aiming to do.

**Comment by danscan** [comment]
Thanks for the link!  FINALLY!

**Comment by madjam002** [comment]
It looks nice from what I've seen, I'm going to try integrating it tonight and decide whether this repo has been made redundant.
